components/org.wso2.carbon.identity.application.authenticator.facebook/pom.xml

```diff
@@ -101,12 +109,17 @@
 
                             org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
                             org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}",
+                            com.nimbusds.jose.*; version="${nimbusds.osgi.version.range}",
```

Comment:
Can we import the specific package?

Reply:
Addressed by https://github.com/wso2-extensions/identity-outbound-auth-facebook/pull/54/commits/edb32edf8233e4325847f5f41ea84e89ffacc40f